content/news/releases/prestashop-8-0-3-maintenance-release.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title:  PrestaShop 8.0.3 is available
+subtitle: Third maintenance version of the 8.0.x branch is available
+date: 2023-03-22
+authors: [ PrestaShop ]
+image: /assets/images/2023/03/banner_8.0.3_1200x627.png
+opengraph_image: /assets/images/2023/03/banner_8.0.3_1200x627.png
+twitter_image: /assets/images/2023/03/banner_8.0.3_1024x512.png
+icon: icon-leaf
+tags:
+- version
+- patch
+- releases
+- "8.0"
+- "8.0.x"
+---
+
+The third patch version for PrestaShop 8 is now available to download!
+
+![8.0.3 is available!](/assets/images/2023/03/banner_8.0.3_1534x424.png)
+
+It was a little bumpy road at the end because of the issues we found during the release process, but here it is! In the newest patch version for PrestaShop 8 you can find [security improvement](https://github.com/PrestaShop/PrestaShop/pull/31647), and three fixes, one related to [inaccurate wording in web services configuration](https://github.com/PrestaShop/PrestaShop/pull/31772), one which [fixes the link to the user documentation in the installer](https://github.com/PrestaShop/PrestaShop/pull/31787), and one [significant fix which improves how clearing the cache behaves](https://github.com/PrestaShop/PrestaShop/pull/31820) in the software.
```

Comment:
```suggestion
It was a rather bumpy road at the end because of the issues we found during the release process, but here it is! In the newest patch version for PrestaShop 8 you can find a [security improvement](https://github.com/PrestaShop/PrestaShop/pull/31647), and three fixes: one related to [inaccurate wording in web services configuration](https://github.com/PrestaShop/PrestaShop/pull/31772), one which [fixes the link to the user documentation in the installer](https://github.com/PrestaShop/PrestaShop/pull/31787), and one [significant fix which improves how clearing the cache behaves](https://github.com/PrestaShop/PrestaShop/pull/31820) in the software.
```